playbook.md

```diff
@@ -77,9 +78,25 @@ However, there are some things that we must keep private.
 - We are sometimes sent documents that are [protectively marked](#protective-marking-scheme), and we have a 
 scheme of our own. Information in these documents is confidential.
 - Information about work that is currently being procured (whether we are bidding or not) and any other information that could damage the commercial interests of a client or supplier
+- Anything to do with the work of dxw cyber: see below.
 
 If you want to talk about something, and you're not sure if it's confidential, play it safe: ask a colleague.
 
+#### Openness and confidentiality (dxw cyber)
+
+dxw cyber's work is usually more sensitive than dxw digital's. We still value openness and believe that it is important, but we have less freedom to talk about our work. Our clients expect us to keep almost everything about our work with them private.
+
+As with dxw digital, we also release [code](http://www.github.com/dxwcyber), publish our [contracts](http://www.github.com/dxw/contracts) and use this playbook. But we must tread with more care, to ensure that we maintain our obligations to clients not to release information that could compromise their systems or cause them embarassment.
+
+As a result, we take the opposite approach to dxw digital. We treat everything about our work as private and confidential, with these exceptions:
+
+- General discussions about tools, techniques, approaches and methodologies
+- The nature of our roles and day-to-day work
+- The purpose, mission, vision and values for dxw cyber, and the ways we get things done
+
+However, even with these exceptions, there is an overriding principle that client information must be kept confidential. For example, if talking about a particular attack technique could allow someone to infer the identity of a client, we would not discuss it in public.
+
+In addition, we take measures to ensure that we don't accidentally expose confidential information. For more information about this, talk to your team lead.
```

Comment:
Who is the 'team lead' here? Might be worth being more explicit, is it a head of? Or a delivery manager?